portfolio/src/main/java/com/google/sps/data/JsonData.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.data;
+
+public class JsonData
+{
+    private final String brand;
+    private final String model;
+    private final String color;
+    private final Integer year;
```

Comment:
When declaring an integer variable, prefer to use the primitive type `int` instead of the wrapper class `Integer`.
```suggestion
    private final int year;
```